packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/LightCurveViewer/LightCurveViewerContainer.js

```diff
@@ -124,16 +124,20 @@ class LightCurveViewerContainer extends Component {
   }
 }
 
-LightCurveViewerContainer.defaultProps = {
-  onReady: () => true
+LightCurveViewerContainer.wrappedComponent.defaultProps = {
+  onReady: () => true,
+  subject: {
```

Comment:
This is different from the default subject prop for the `SingleImageViewerContainer`.  Subject file viewers should probably be consistent in their props, so we should decide whether that component should set a default subject too.

Reply:
The LCV container is using subject id in the componentDidUpdate lifecycle method, so we need a default. We could have a default empty string for id for all the viewers and check that?

Reply:
I think I'd prefer if the default subject is null, but I'm not sure. Is there a reason for having a mock subject as the default? It seems to me that this means any code that handles the subject must first check if it's real and not the default.

Reply:
I think it makes sense to mock the subject because there _should_ be a subject by the time the specific subject viewer component loads. If there isn't, then something is really going wrong with the parent component or the store. 

We could instead set the subject prop to be required and remove the default and check for it being undefined in `componentDidUpdate`. I don't have a strong opinion either way, but this component shouldn't be mounting without a subject.

Reply:
Good point. I was thinking of the case where the default subject gets passed down the tree, but maybe that never happens.